Common/UnitDefinitions/Volume.json

```diff
@@ -210,8 +210,8 @@
     {
       "SingularName": "ImperialGallon",
       "PluralName": "ImperialGallons",
-      "FromUnitToBaseFunc": "{x} * 0.00454609000000181429905810072407",
-      "FromBaseToUnitFunc": "{x} / 0.00454609000000181429905810072407",
+      "FromUnitToBaseFunc": "{x} * 0.00454609",
+      "FromBaseToUnitFunc": "{x} / 0.00454609",
       "Prefixes": [ "Kilo", "Mega" ],
```

Comment:
While touching this, can you add a very brief description in summary and a link to the source conversion factor in the remarks of the unit? For both units.

Example:
```json
      "XmlDocSummary": "The US liquid gallon (frequently called simply \"gallon\") is legally defined as 231 cubic inches, which is exactly 3.785411784 litres.",
      "XmlDocRemarks": "https://en.wikipedia.org/wiki/Gallon#US_liquid_gallon",
```